platform/integrations/ex-integration-guide/exclient-upload-recordings.js

```diff
@@ -47,6 +49,27 @@ async function uploadRecordings(fileName) {
         let response = await client.loginClientCredentialsGrant(clientId, clientSecret);
         logger.verbose('Login successfully');
 // >> START ex-get-presigned-url
+
```

Comment:
In the comments above that explain the steps (lines 40 to 44) we need to add a line about zipping the recording file with the metadata file